members.php

```diff
@@ -442,23 +441,66 @@ function members() {
 			case "highest_exp":
 				$table_header = 'Experience';
 				break;
-			case "highest_pvp":
-				$table_header = 'Pvp Kills';
-				break;
 			case "online_users":
 				$table_header = 'Level';
 				break;
+				case "highest_pvp":
+					$table_header = 'Pvp Kills';
+					break;
+				case "highest_teams":
+					$table_header = 'Highest Teams';
+					break;
 		}
 
 		// Search box for individual users
-		// List top 10 users by experience
-		echo "<table class='table'><tr><th colspan='4'>$list_name</th></tr>
-		<tr>
-			<th style='width:30%;'>Username</th>
-			<th style='width:20%;'>Rank</th>
-			<th style='width:20%;'>Village</th>
-			<th style='width:30%;'>" . ($table_header) . "</th>
-		</tr>";
+		// list top 15 Teams
+		if($view == 'highest_teams') {
+			// Teams
+				$user_id_array = array();
+				$result = $system->query("SELECT * FROM `teams` ORDER BY `monthly_points` DESC LIMIT 15");
+				$teams = array();
+				while($row = $system->db_fetch($result)) {
+					$teams[] = $row;
+					if(array_search($row['leader'], $user_id_array) === false) {
+							$user_id_array[] = $row['leader'];
+						}
+					}
+				// Fetch leader names
+				$user_id_string = implode(',', $user_id_array);
+				$result = $system->query("SELECT `user_id`, `user_name`, `village` FROM `users` WHERE `user_id` IN ($user_id_string)");

```

Comment:
This will error if `$user_id_string` is blank, we should skip running this query if `$user_id_array` is empty